docs/features/multiple-networks/multi-network-policies.md

```diff
@@ -1,20 +1,43 @@
 # MultiNetworkPolicies
-OVN-Kubernetes implements native support for
-[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy),
-an API providing
+## Introduction
+[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy) provides
 [network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
-features for secondary networks.
+features for secondary networks, and allow enhanced traffic security.
 
+## Motivation
+In Kubernetes the paradigm is that by default all pods can reach other pods, and security is provided by
+implementing [Network Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
+MultiNetworkPolicies provide the same security solution to secondary interfaces.
+For users seeking network security using multi-homing - traffic restrictions based on user network configuration is a
+recommended practice.
+
+For example, by defining policies that specify which services a workload can access, they mitigate supply chain attacks.
+Even if a component is compromised, it won’t be able to jeopardize other components of the system.
+
+## User-Stories
+As a project administrator, I want to enforce network policies that must be adhered to by all workloads on
+my namespace. This will secure my namespace’s network traffic.
+
+## User-case
```

Comment:
there are many more; I think we should add one that makes sense for flat layer2.

Reply:
OK I added a flat layer2 one. tell me what you think